application/management/commands/helsinki_activities_test_data.py

```diff
@@ -9,49 +9,35 @@
 
 COLLECTION_BASE = {
     "visible_on_frontpage": True,
-
     "box_color": "SUOMENLINNA",
-
     "curated_events": [
-        ('event_link', "http://localhost:3000/fi/event/helsinki:afxh3naida?id=123"),
-        ('event_link', "http://localhost:3000/fi/event/helsinki:afxrsql3xa"),
-        ('event_link', "http://localhost:3000/fi/event/helsinki:afxh3namhe"),
-        ('event_link', "http://localhost:3000/fi/event/helsinki:afxpj6bxbu"),
-        ('event_link', "http://localhost:3000/fi/event/helsinki:afx5msunhu"),
+        ("event_link", "http://localhost:3000/fi/event/helsinki:afxh3naida?id=123"),
+        ("event_link", "http://localhost:3000/fi/event/helsinki:afxrsql3xa"),
+        ("event_link", "http://localhost:3000/fi/event/helsinki:afxh3namhe"),
+        ("event_link", "http://localhost:3000/fi/event/helsinki:afxpj6bxbu"),
+        ("event_link", "http://localhost:3000/fi/event/helsinki:afx5msunhu"),
     ],
-
     "curated_events_title_en": "At least visit these",
     "curated_events_title_fi": "Käy ainakin näissä",
     "curated_events_title_sv": "Besök åtminstone dessa",
-
     "description_en": "Etiam rhoncus. Maecenas tempus, tellus eget condimentum rhoncus, sem quam semper libero",
     "description_fi": "Etiam rhoncus. Maecenas tempus, tellus eget condimentum rhoncus, sem quam semper libero",
     "description_sv": "Etiam rhoncus. Maecenas tempus, tellus eget condimentum rhoncus, sem quam semper libero",
-
     "social_media_description_en": "social media is social. eget condimentum rhoncus, sem quam semper libero",
     "social_media_description_fi": "social media is social. eget condimentum rhoncus, sem quam semper libero",
     "social_media_description_sv": "social media is social. eget condimentum rhoncus, sem quam semper libero",
-
     "event_list_query_fi": "http://localhost:3000/fi/events?categories=music&districts=kaupunginosa%3Aetu-%B6l%C3%B6",
     "event_list_query_sv": "http://localhost:3000/sv/events?categories=music&districts=kaupunginosa%3Aetu-%B6l%C3%B6",
     "event_list_query_en": "http://localhost:3000/en/events?categories=music&districts=kaupunginosa%3Aetu-%B6l%C3%B6",
-
-    "course_list_query_fi": "http://localhost:3000/fi/courses?categories=music&districts=kaupunginosa%3Aetu-%B6l%C3%B6",
-    "course_list_query_sv": "http://localhost:3000/sv/courses?categories=music&districts=kaupunginosa%3Aetu-%B6l%C3%B6",
-    "course_list_query_en": "http://localhost:3000/en/courses?categories=music&districts=kaupunginosa%3Aetu-%B6l%C3%B6",
-
```

Comment:
This is actually the only part t hat has changed in the test file. Other changes here are done by autoformatter.